app/views/layouts/mailer.html.erb

```diff
@@ -3,11 +3,41 @@
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <style>
-      /* Email styles need to be inline */
+      * {
+        font-family: Helvetica Nue, Helvetica, Arial, sans-serif;
+      }
+      .wrapper {
+        max-width: 500px;
+        margin: 0 auto;
+      }
+      .wrapper p {
+        margin: 1.5rem 0;
+      }
+      .logo {
+        margin: 2.5rem 0;
+      }
+      .logo img {
+        width: 180px;
+        height: auto;
+      }
+      .subject {
+        font-size: 20px;
+        color: #757575;
+        font-weight: normal;
+        margin-bottom: 2.5rem;
+      }
     </style>
   </head>
 
   <body>
-    <%= yield %>
+    <div class="wrapper">
+      <div class="logo">
+        <%= image_tag "GYR_logo.png", style: "width:180px;height:auto" %>
```

Comment:
Would it be OK to pick one of adding inline style here _or_ adding the image style to the CSS? If the key is to always style this image, even in the face of abysmal email clients which don't support `<style>`, then I'd say pick only the inline style. Not a blocker.

Reply:
done! left inline to ensure it doesnt end up huge on a wonky client.